tasks/create-vm/manifests/create-vm-from-template.yaml

```diff
@@ -39,6 +41,14 @@ spec:
       description: Namespace where to create the VM. (defaults to active namespace)
       default: ""
       type: string
+    - name: sshAuthorizedKeysSecret
```

Comment:
The disadvantage is that the config is quite verbose since we have to specify the users too (possibly propagation police), but the execute-in-vm task already knows the user.

Reply:
I would vote to add it only to create-vm-from-template - it makes maintenance of the pipelines easier if one thing is only done one way. Adding it to the other tasks would be useful only if I wanted to have a flow where I take an existing VM and add new keys into it and I dont see that as a valid use case.